schema/iguana.owl

```diff
@@ -1,4 +1,3 @@
-
 <!DOCTYPE rdf:RDF [
     <!ENTITY dcterms "http://purl.org/dc/terms/" >
     <!ENTITY owl "http://www.w3.org/2002/07/owl#" >
```

Comment:
Please double check what a good ontology name would be. This one with the owl file seems not ideal to me. 

Something like `https://iguana-benchmark.eu/ontology/` or even https://vocab.dice-research.org/iguana/` seems to fit better. It would also make sense to have a version iri like `https://vocab.dice-research.org/iguana/4.0.0/`

Also, `iont`, `iprop` and `ires` should be relative to that.

If we define the ontology to live under dice it should be published there. That is a follow up task. Please contact @lennartaustenfeld about that.

Reply:
I change the name to `https://iguana-benchmark.eu/ontology/` for now and added the version iri `https://vocab.dice-research.org/iguana/4.0.0/`.

I made `iont`, `iprop` and `ires` relative to `https://iguana-benchmark.eu/ontology/`, i don't know if that's correct.

Reply:
I've also created an issue for possible renaming and for making the ontology available online.